src/dialogs/more-info-dialog.html

```diff
@@ -19,6 +19,7 @@
       }
 
       state-history-charts {
+        width: 100%;
```

Comment:
@balloob I didn't think so, but I am having trouble replicating the issue. It seems very intermittent. I'm thinking we should specify specific widths using media queries.